modules/exploits/linux/http/trendmicro_websecurity_exec.rb

```diff
@@ -0,0 +1,167 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Trend Micro Web Security (Virtual Appliance) Remote Code Execution',
+      'Description'    => %q{
+        This module exploits multiple vulnerabilities together in order to achive a remote code execution.
+        Unauthenticated users can execute a terminal command under the context of the root user.
+
+        The specific flaw exists within the LogSettingHandler class of administrator interface software.
+        When parsing the mount_device parameter, the process does not properly validate a user-supplied string
+        before using it to execute a system call. An attacker can leverage this vulnerability to execute code in
+        the context of root. But authentication is required to exploit this vulnerability.
+
+        Another specific flaw exist within the proxy service, which listens on port 8080 by default. Unauthenticated users
+        can exploit this vulnerability in order to communicate with internal services in the product.
+
+        Last but not least flaw exists within the Apache Solr application, which is installed within the product.
+        When parsing the file parameter, the process does not properly validate a user-supplied path prior to using it in file operations.
+        An attacker can leverage this vulnerability to disclose information in the context of IWSS user.
+
+        Due to combination of these vulnerabilities, unauthenticated users can execute a terminal command under the context of the root user.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Mehmet Ince <mehmet@mehmetince.net>' # discovery & msf module
+        ],
+      'References'     =>
+        [
+          ['CVE', '2020-8604'],
+          ['CVE', '2020-8605'],
+          ['CVE', '2020-8606'],
+          ['ZDI', '20-676'],
+          ['ZDI', '20-677'],
+          ['ZDI', '20-678']
+        ],
+      'Privileged'     => true,
+      'DefaultOptions' =>
```

Comment:
bind payloads dont seem to be working.  Consider adding something to remove them from the payload list: https://github.com/rapid7/metasploit-framework/blob/76954957c740525cff2db5a60bcf936b4ee06c42/modules/exploits/unix/http/vmturbo_vmtadmin_exec_noauth.rb#L45

Reply:
Ahh yeah, my bad sorry <3 .